drake/systems/plants/RigidBodyTree.h

```diff
@@ -416,26 +412,6 @@ class DLLEXPORT_RBM RigidBodyTree
   */
   
   template <typename Derived>
-Eigen::Matrix<typename Derived::Scalar, Derived::RowsAtCompileTime, Eigen::Dynamic> transformPositionDotMappingToVelocityMapping(
-    const KinematicsCache<typename Derived::Scalar>& cache, const Eigen::MatrixBase<Derived>& mat) const
-{
-  Eigen::Matrix<typename Derived::Scalar, Derived::RowsAtCompileTime, Eigen::Dynamic> ret(mat.rows(), num_velocities);
-  int ret_col_start = 0;
-  int mat_col_start = 0;
-  for (auto it = bodies.begin(); it != bodies.end(); ++it) {
-    RigidBody& body = **it;
-    if (body.hasParent()) {
-      const DrakeJoint& joint = body.getJoint();
-      const auto& element = cache.getElement(body);
-      ret.middleCols(ret_col_start, joint.getNumVelocities()).noalias() = mat.middleCols(mat_col_start, joint.getNumPositions()) * element.v_to_qdot;
-      ret_col_start += joint.getNumVelocities();
-      mat_col_start += joint.getNumPositions();
-    }
-  }
-  return ret;
-};
-
-  template <typename Derived>
```

Comment:
transformPositionDotMappingToVelocityMapping and its inverse have been moved from RigidBodyTree to KinematicsCache